ui/app/AppMain.qml

```diff
@@ -35,7 +35,7 @@ Item {
     // set from main.qml
     property var sysPalette
     // Not Refactored Yet
-    property var newVersionJSON: "" //JSON.parse(utilsModel.newVersion)
+    property var newVersionJSON: ({}) //JSON.parse(utilsModel.newVersion)
```

Comment:
@jrainville as a part of this ticket we should move code which is currently placed in the file `src/app/REMOVE-utilsView/view` to the `mainModule` (`QtProperty[string] newVersion` should be part of the `src/app/modules/main/view.nim`) and used in this line like: 
```
property var newVersionJSON: mainModule.newVersion
```

Async task `CheckForNewVersionTaskArg` for fetching new version from remote may be part of `about` service (if you want you may add kind of a `global` service, but I think having this as a part of `about` service is just fine) and have `getNewAppVersion` method, and you may just call it from a `controller` of the main module.